src/tests/ftest/util/ec_utils.py

```diff
@@ -73,8 +109,14 @@ def ior_write_dataset(self):
                 if oclass[1] > self.server_count:
                     continue
                 self.ior_param_update(oclass, sizes)
-                self.update_ior_cmd_with_pool(oclass=oclass[0])
+
+                #Create the new container with correct redundancy factor
+                #for EC object type
+                self.ec_contaier_create(oclass[0])
```

Comment:
Is oclass a list? If so, why only using oclass[0].

Reply:
Yes, it's a list with a Minimum Number of servers supported by that OBJ class. oclass[1] is used to check the count of servers to avoided writing if the object type is not supported.
The logic behind this is if we have 4 servers we need to only write 2+1 or 2+2 but if we have 10 servers we need to write all objects from a minimum 2+1,2+2,4+1,4+2,8+2. 

```
    #- [EC_Object_Class, Minimum number of servers]
    - ["EC_2P1G1", 4]
```